commitlint.config.js

```diff
@@ -13,6 +13,7 @@ function getPackageAbbreviations() {
     .map(pkg => pkg.name)
     .map(name => (name.charAt(0) === '@' ? name.split('/')[1] : name))
     .map(name => (name.includes(prefix) ? name.replace(prefix, '') : name))
+    .concat(['project'])
```

Comment:
i added a new `project` scope for commitizen so we can do things outside the packages